tests/DCPS/XTypes/run_test.pl

```diff
@@ -23,7 +25,20 @@
   push(@common_args, "--verbose");
 }
 
-my %params = (
+my %params;
+if ($secure) {
+%params = (
+  "FinalStructMatch_s"        => {reader_type => "FinalStructSub", writer_type => "FinalStructPub", expect_to_fail => 0, reg_type => "PD_OL_OA_OM_OD", key_val => 2,
+                                r_ini => "rtps_disc_security.ini", w_ini => "rtps_disc_security.ini"},
```

Comment:
Indent is off here and on the other secure line.  Also these lines are kinda long. Personally I'd format them all like this:

```perl
    "FinalStructMatch_s" => {
      reader_type => "FinalStructSub", writer_type => "FinalStructPub",
      expect_to_fail => 0, reg_type => "PD_OL_OA_OM_OD", key_val => 2,
      r_ini => "rtps_disc_security.ini", w_ini => "rtps_disc_security.ini"
    },
```